python-frontend/src/main/java/org/sonar/python/types/v2/ModuleType.java

```diff
@@ -30,23 +30,23 @@
 public final class ModuleType implements PythonType {
   private final String name;
   private final ModuleType parent;
-  private final Map<String, PythonType> members;
+  private final Map<String, TypeWrapper> members;
+  private final Map<String, TypeWrapper> subModules = new HashMap<>();
```

Comment:
I'd suggest to assign the field value in the same place where other fields are initialized, to have single place to look at